interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -128,6 +128,32 @@ function ViewModel() {
         return parseInt(self.nrWarnings()) + self.allMessages().length;
     })
 
+    // process the full queue
+    self.processFullQueue = function(response){
```

Comment:
This was more a i want to guarentee i get the actual first and last element and kind of a preemptive attempt at finding the first and last that were not paused.  It is not 100% necessary but gives a place for logic to happen on queue position selection if necessary.